app/src/main/java/com/xmartlabs/daydreaming/ui/DiagonalLayoutView.java

```diff
@@ -53,5 +59,45 @@ private void setupViewProperties(AttributeSet attrs) {
   public void setTextColor(@ColorRes int color) {
     //noinspection deprecation
     textView.setTextColor(getResources().getColor(color));
+    //noinspection deprecation
+    subtitleView.setTextColor(getResources().getColor(color));
+  }
+
+  public void setTextSize(@Dimension(unit = Dimension.SP) float size) {
+    textView.setTextSize(Dimension.SP, size);
+    subtitleView.setTextSize(Dimension.SP, size);
+    secondaryTextView.setTextSize(Dimension.SP, size);
+  }
+
+  public void setDiagonalImageAlpha(int alpha) {
+    imageView.setImageAlpha(alpha);
+  }
+
+  public void setSubtitleText(@StringRes int text) {
+    subtitleView.setText(text);
+  }
+
+  public void setSubtitleText(String text) {
+    subtitleView.setText(text);
+  }
+
+  public void setSubtitleVisible() {
```

Comment:
I'd say something like showSubtitle or hideSubtitle or stuff.
Set makes me think like an attribute of the object itself.
Just my opinion